### DISCUSSION TITLE
45˚ Profile 144leds/m Optimum Distance

### DISCUSSION THREAD

**Discussion by dizzwold** [awawa-dev/HyperHDR#948] [General]
Hi Guys,

 This is my first build, so please excuse me if sounding daft.

 I'm really grateful for this project and the work and effort that's been put into creating such.

 I have some SK6812 CW 144LED's/m. What would be the recommended/optimum distance away from the wall on 45˚profile?

 I currently have 50mm to the back wall of the profile when in position

 I have a maximum distance of 70mm, but as the TV is on a full motion bracket It could be pulled out a little to create extra spacing.
 Failing that, I do also have some flat profile.

  When I powered up the project a few months ago, just for testing the PI, Scorpio Board and a few LEDs, am I correct in remembering that the starting Data Line LED was along the top row, with the choice of choosing either left side or right?

 Would I be correct in stating I should add Power Injection roughly every meter with using 144LEDs/m?

 I'd be grateful for you advice.

**Comment by awawa-dev** [comment]
Hi
The closer the better, LEDs emit the most light within a 60 degree angle, so if they are far away (even a few cm away can be too much for 144LEDs/meter) the colors on the wall will blend. It's hard to get an exact result and you have to test it yourself to see what suits you best.

**Comment by dizzwold** [comment]
Hi @awawa-dev 

 Thank you for your reply. I'll experiment and play once I get the LEDs wired-up.
![IMG_3708](https://github.com/user-attachments/assets/a17e9700-742c-415f-8db6-61df289ec9e3)
![IMG_3710](https://github.com/user-attachments/assets/0fd20fe8-cd5d-4934-8af8-14f4e3e6d8a9)
![IMG_3711](https://github.com/user-attachments/assets/f0637edf-2eaf-4ccf-b5af-828fea5effa8)
